modules/products/pycharm/src/main/java/net/ashald/envfile/products/pycharm/PyCharmRunConfigurationExtension.java

```diff
@@ -53,8 +53,8 @@ protected void validateConfiguration(@NotNull AbstractPythonRunConfiguration con
 
     @Override
     protected void patchCommandLine(@NotNull AbstractPythonRunConfiguration configuration, @Nullable RunnerSettings runnerSettings, @NotNull GeneralCommandLine cmdLine, @NotNull String runnerId) throws ExecutionException {
+        Map<String, String> newEnv = EnvFileConfigurationEditor.collectEnv(configuration, EnvUtil.getInitialEnv(cmdLine));
         Map<String, String> currentEnv = cmdLine.getEnvironment();
-        Map<String, String> newEnv = EnvFileConfigurationEditor.collectEnv(configuration, new HashMap<>(currentEnv));
```

Comment:
After this change, `currentEnv ` defined above becomes unused - should we drop it altogether?

Reply:
As above. Keeping the references to `currentEnv` together seemed clearer to me.